docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -1,234 +0,0 @@
-----
-
-. Enable root access for the *docker-machine-driver-xhyve* binary and add it to the default *wheel* group:
-+
-----
-$ sudo chown root:wheel /usr/local/bin/docker-machine-driver-xhyve
-----
-
-. Set owner User ID (SUID) for the binary as follows:
-+
-----
-$ sudo chmod u+s,+x /usr/local/bin/docker-machine-driver-xhyve
-----
-
-[NOTE]
-====
-The downloaded *docker-machine-driver-xhyve* binary is compiled against a specific version of macOS.
-It is possible that the driver will fail to work after a macOS version upgrade.
-In this case you can try to compile the driver from source:
-
-----
-$ go get -u -d github.com/zchee/docker-machine-driver-xhyve
-$ cd $GOPATH/src/github.com/zchee/docker-machine-driver-xhyve
-
-# Install docker-machine-driver-xhyve binary into /usr/local/bin
-$ make install
-
-# docker-machine-driver-xhyve need root owner and uid
-$ sudo chown root:wheel /usr/local/bin/docker-machine-driver-xhyve
-$ sudo chmod u+s /usr/local/bin/docker-machine-driver-xhyve
-----
-
-For more information, see the link:https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
-====
-
-== Hyper-V Driver
-To use {project} with Hyper-V:
-
-. Install link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V].
-
-. Add the user to the local Hyper-V Administrators group.
-+
-[NOTE]
-====
-This is required to allow the user to create and delete virtual machines with the Hyper-V Management API.
-For more information, see xref:../troubleshooting/troubleshooting-driver-plugins.adoc#insufficient-privileges[Hyper-V commands must be run as an Administrator].
-====
-
-. Add an link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[External Virtual Switch].
-+
-[NOTE]
-====
-Verify that you pair the virtual switch with a network card (wired or wireless) that is connected to the network.
-====
-
-. Set the environment variable `HYPERV_VIRTUAL_SWITCH` to the name of the external virtual switch you want to use for {project}.
-+
-For example, on Command Prompt use:
-+
-----
-C:\> set HYPERV_VIRTUAL_SWITCH=External (Wireless)
-----
-+
-Note that using quotes in Command Prompt results in the following error:
-+
-----
-C:\> set HYPERV_VIRTUAL_SWITCH="External (Wireless)"
-Error creating the VM. Error with pre-create check: "vswitch \"\\\"External (Wireless)\\\"\" not found"
-----
-+
-However, on PowerShell you need to use the quotes:
-+
-----
-PS> $env:HYPERV_VIRTUAL_SWITCH="External (Wireless)"
-----
-+
-Using `set` in Command Prompt or `$env` in PowerShell will only affect the currently running shell session.
-You may use the `setx` utility to persist this environment variable across shell instances and system restarts.
-Quotes must be used with `setx` for both Command Prompt and PowerShell.
-For example:
-+
-----
-C:\> setx HYPERV_VIRTUAL_SWITCH "External (Wireless)"
-----
```

Comment:
This needs code changes in `cmd/minishift/cmd/start_preflight.go :   driverErrorMessage := "See the 'Setting Up the Driver Plug-in' topic (https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html) for more information"`